mentorship ios/Views/Home/Home.swift

```diff
@@ -35,6 +36,24 @@ struct Home: View {
         }
     }
     
+    func getUserFirstName(fullName: String?) -> String {
+         
+        //Return just the first name
+        if let editFullName = fullName ?? "" {
+            
+            //Field validation
+            //get rid of any spaces before the beginning of the name
+            let trimmedFullName = editFullName.trimmingCharacters(in: .whitespaces)
+            
+            if let index = trimmedFullName.firstIndex(of: " ") {
+                let firstName = String(trimmedFullName.prefix(upTo: index))
+                return firstName
+            }
+        }
+         
+        return ""
+    }
+    
```

Comment:
1. Consider using .split method.
2. Consider making a computed property for full name instead of a function.

Reply:
Ah yes, that is the better way. Thanks for the guidance and feedback.